moto/cloudformation/responses.py

```diff
@@ -77,6 +77,50 @@ def create_stack(self):
             template = self.response_template(CREATE_STACK_RESPONSE_TEMPLATE)
             return template.render(stack=stack)
 
+    def create_change_set(self):
+        stack_name = self._get_param('StackName')
+        change_set_name = self._get_param('ChangeSetName')
+        stack_body = self._get_param('TemplateBody')
+        template_url = self._get_param('TemplateURL')
+        role_arn = self._get_param('RoleARN')
+        update_or_create = self._get_param('ChangeSetType', 'CREATE')
+        parameters_list = self._get_list_prefix("Parameters.member")
+        tags = dict((item['key'], item['value'])
+                    for item in self._get_list_prefix("Tags.member"))
+        # Hack dict-comprehension
```

Comment:
There a reason why we cant use regular dictionary comprehensions?

Reply:
I was just following convention in the file.  I would be happy to change it to a real dict comprehension.

Reply:
Yeah swap it out, we dont need to support py26 anymore and py27 supports it if I remember correctly.

Reply:
Yea - that's what I was thinking - probably some 2.6 cruft.